src/renderer/components/confirm-dialog/confirm-dialog.scss

```diff
@@ -33,7 +33,8 @@
     max-width: 50vw;
     min-width: 45 * $unit;
     background-color: white;
-    outline: $unit solid rgba(255, 255, 255, .15);
+    border-radius: $radius;
```

Comment:
Some fine-tunings in dialog-related styles to keep all dialogs views consistent (rounded a bit).